Doc/library/sqlite3.rst

```diff
@@ -844,7 +844,7 @@ Cursor Objects
       :ref:`placeholders <sqlite3-placeholders>`.
 
       :meth:`execute` will only execute a single SQL statement. If you try to execute
-      more than one statement with it, it will raise a :exc:`ProgrammingError`. Use
+      more than one statement with it, it will raise a :exc:`.Warning`. Use
```

Comment:
Is this valid ReST? Shouldn't it be :exc:`Warning`?

Reply:
Outdated :)

Reply:
FTR; this is how it was in the first place. I haven't seen the linter complain about it before, so I guess it is valid. But, I reverted this change in 8b7293e, so it does not matter anymore :)